main.go

```diff
@@ -140,6 +143,13 @@ func main() {
 		os.Exit(0)
 	}
 
+	// Ensure we delete the work item cache when we receive a notification from postgres
+	gormsupport.SetupDatabaseListener(*config, map[string]gormsupport.SubscriberFunc{
```

Comment:
What happens when `SetupDatabaseListener` function return an error? The error value is not handled here.

Reply:
Good catch. I will have a look when this PR becomes more relevant.